hammer_cli_foreman_leapp.gemspec

```diff
@@ -9,11 +9,10 @@ require 'hammer_cli_foreman_leapp/version'
 Gem::Specification.new do |spec|
   spec.name          = 'hammer_cli_foreman_leapp'
   spec.version       = HammerCLIForemanLeapp.version.dup
-  spec.license       = 'GPL-3.0'
-  spec.date          = Time.now.utc.to_date.to_s
+  spec.license       = 'GPL-3.0-only'
   spec.authors       = ['Foreman Leapp team']
   spec.email         = ['foreman-dev@googlegroups.com']
-  spec.homepage      = 'https://github.com/stejskalleos/hammer-cli-foreman-leapp'
+  spec.homepage      = 'https://github.com/theforeman/hammer-cli-foreman-leapp'
```

Comment:
Should we also move the repo so it matches the gem name?

Reply:
Sure.